src/main/kotlin/io/github/anki/anki/service/exceptions/DoesNotExist.kt

```diff
@@ -0,0 +1,6 @@
+package io.github.anki.anki.service.exceptions
+
+interface DoesNotExist {
```

Comment:
why you need this?

Reply:
because i want to have 1 method in ExceptionHandler to receive DoesNotExist type and return user 400